mk/dev.mk

```diff
@@ -1,7 +1,11 @@
 KUMA_DIR ?= .
 ENVOY_VERSION = $(word 5, $(shell ${KUMA_DIR}/tools/releases/version.sh))
+CI_TOOLS_VERSION = $(word 6, $(shell ${KUMA_DIR}/tools/releases/version.sh))
+KUMA_CHARTS_URL ?= https://kumahq.github.io/charts
+CHART_REPO_NAME ?= kuma
+PROJECT_NAME ?= kuma
 
-CI_TOOLS_DIR ?= ${HOME}/.kuma-dev
+CI_TOOLS_DIR ?= ${HOME}/.kuma-dev/${PROJECT_NAME}-${CI_TOOLS_VERSION}
```

Comment:
`CI_TOOLS_VERSION` here is causing https://app.circleci.com/pipelines/github/Kong/kong-mesh/7791/workflows/789b6774-724b-4291-b3a2-944bcc26e622/jobs/71159?invite=true#step-104-6 because ci tools checksum can be equal on multiple branches and when it is we have a mismatch of a folder. Possible solutions:
1. use `${PROJECT_NAME}-${CI_TOOLS_CHECKSUM}`
2. use `- golang_go.mod_{{ checksum "go.sum" }}_{{ checksum "mk/dependencies/deps.lock" }}_{{ checksum ".circleci/config.yml" }}_${CI_TOOLS_VERSION}`

1. Should be good, would be my preferred option
2. Probably would work too, but now sure how easy would be to call a bash script in `restore_cache` or pass it there somehow.

WDYT?

Reply:
I think I would go with the first point